helm/netobserv/templates/deployment.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wnetobserv-controller-manager
+  labels:
+    app: netobserv-operator
+    control-plane: controller-manager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: netobserv-operator
+      control-plane: controller-manager
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: netobserv-operator
+        control-plane: controller-manager
+    spec:
+      containers:
+      - args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
+        - --leader-elect
+        - --ebpf-agent-image=$(RELATED_IMAGE_EBPF_AGENT)
+        - --flowlogs-pipeline-image=$(RELATED_IMAGE_FLOWLOGS_PIPELINE)
+        - --console-plugin-image=$(RELATED_IMAGE_CONSOLE_PLUGIN)
+        - --downstream-deployment=$(DOWNSTREAM_DEPLOYMENT)
+        - --profiling-bind-address=$(PROFILING_BIND_ADDRESS)
+      command:
```

Comment:
there's a missing indent issue starting here (`command`) til the end of the file